### PR TITLE
Speed up getting latest tvl

### PR DESCRIPTION
**Pull request by torztomasz**

### CODE REVIEW

File: packages/database/src/tvl/value/select.ts [hunk 1]
```diff
@@ -19,3 +19,6 @@ export const selectValue = [
   'ether',
   'stablecoin',
 ] as const satisfies (keyof Value)[]
+
+export const selectValueWithPrefix = <T extends string>(prefix: T) =>
```
Comment (sdlyy) [inline packages/database/src/tvl/value/select.ts:23]: Is it used anywhere?

File: packages/database/src/tvl/value/repository.ts [hunk 2]
```diff
@@ -46,6 +48,40 @@ export class ValueRepository extends BaseRepository {
...
+
+    const rows = await this.db
+      .with('latest_values', latestValuesQuery)
+      .selectFrom('latest_values')
+      .select(selectValue)
+      .where('combination_number', '=', 1)
+      .execute()
+    return rows.map(toRecord)
+  }
+
```
Comment (sdlyy) [inline packages/database/src/tvl/value/repository.ts:84]: We have almost 1:1 duplicate down below for da-beat - Let's either unify these or split this apart removing the optional param (`projectIds?: ProjectId[]`)

File: packages/frontend2/src/app/(new)/(other)/scaling/data-availability/page.tsx [hunk 3]
```diff
@@ -12,8 +11,7 @@ export const metadata = getDefaultMetadata({
 })
 
 export default async function Page() {
-  const tvl = await getLatestTvlUsd({ type: 'all' })
-  const items = await getScalingDaEntries(tvl)
+  const items = await getScalingDaEntries()
```
Comment (sdlyy) [inline packages/frontend2/src/app/(new)/(other)/scaling/data-availability/page.tsx:14]: Nit: Let's unify this and go with `entries` as well

**Review by sdlyy — CHANGES_REQUESTED** [review]

File: packages/database/src/tvl/value/select.ts (hunk 1 above)
Comment (torztomasz) [reply packages/database/src/tvl/value/select.ts:23]: I was using it, forgot to remove it

File: packages/database/src/tvl/value/repository.ts (hunk 2 above)
Comment (torztomasz) [reply packages/database/src/tvl/value/repository.ts:84]: yup, i wanted to unify that

**Review by sdlyy — APPROVED** [review]

**Pull request merged**